src/TSTransformer/util/hasMultipleDefinitions.ts

```diff
@@ -0,0 +1,28 @@
+import { TransformState } from "TSTransformer";
+import { addDiagnosticIfNotCached } from "TSTransformer/util/addDiagnosticIfNotCached";
+import ts from "typescript";
+
+export function hasMultipleDeclarations(
+	state: TransformState,
+	symbol: ts.Symbol,
+	filter: (declaration: ts.Declaration) => boolean,
+	error?: ts.Diagnostic,
```

Comment:
error reporting should be done at the call site, not here

Reply:
For more code duplication at both of the callsites? That doesn't make much sense to me

Reply:
It's very little code duplication and shows the intent clearer. Rather than having to figure out what this magic argument does within this function call.

Reply:
You're right, changed.